100_essentials/greetings/README.md

```diff
@@ -0,0 +1,49 @@
+# Greetings
+
+Everything you need to complete this challenge can be found [here](https://docs.python.org/3.8/tutorial/controlflow.html#defining-functions).
+
+---
+
+Define a function named `greetings` that takes a required positional argument `name` and a required [key word only](https://docs.python.org/3/glossary.html#term-parameter) argument `punctuation` and returns a greeting in the following format:
+
```

Comment:
This is _much_ better. Great challenge for learning functions. I have no feedback on this at all.